launcher/settingsView/csettingsview_moc.ui

```diff
@@ -1449,6 +1449,35 @@ Fullscreen Exclusive Mode - game will cover entirety of your screen and will use
          </attribute>
         </widget>
        </item>
+       <item row="61" column="0">
+        <widget class="QLabel" name="labelPreLauncher">
+         <property name="text">
+          <string>Prelauncher</string>
```

Comment:
Prelauncher is very unclear IMO.
Not sure what to call it though. If somebody has better suggestions than this - go ahead.
```suggestion
          <string>Skip Launcher on start</string>
```